pkg/operator/reporting/templates_test.go

```diff
@@ -0,0 +1,569 @@
+package reporting
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/operator-framework/operator-metering/pkg/operator/prestostore"
+
+	"github.com/operator-framework/operator-metering/pkg/apis/metering/v1alpha1"
+	metering "github.com/operator-framework/operator-metering/pkg/apis/metering/v1alpha1"
+	"github.com/operator-framework/operator-metering/pkg/presto"
+	"github.com/operator-framework/operator-metering/test/testhelpers"
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestRenderQuery(t *testing.T) {
+	const (
+		testNamespace            = "default"
+		testReportName           = "unready-deployment-replicas"
+		testReportQuery          = "unready-deployment-replicas"
```

Comment:
Maybe make `testReportName` and `testReportQuery` have distinct values so the possibility of mixing them up would cause an error.